### PR TITLE
Add JS grammar compat data

### PR DESCRIPTION
**Pull request by Elchi3**
Compat data for 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

### CODE REVIEW

File: javascript/grammar.json [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+              "version_added": "30"
+            },
+            "opera_android": {
+              "version_added": "30"
+            },
+            "safari": {
+              "version_added": "9"
+            },
+            "safari_ios": {
+              "version_added": false
```
Comment (teoli2003) [inline javascript/grammar.json]: This one is strange. I think it should be null.

File: javascript/grammar.json [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+              "version_added": "28"
+            },
+            "opera_android": {
+              "version_added": "28"
+            },
+            "safari": {
+              "version_added": "9"
+            },
+            "safari_ios": {
+              "version_added": false
```
Comment (teoli2003) [inline javascript/grammar.json]: I would prefer null, too.

File: javascript/grammar.json [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "template_literal_revision": {
+          "__compat": {
+            "description": "Template literal revision",
```
Comment (teoli2003) [inline javascript/grammar.json]: I think this description is unhelpful, we should have a brief description of the actual change.
What about: "escape sequences allowed on tagged template literals" (or something shorter)

File: javascript/grammar.json [hunk 4]
```diff
@@ -0,0 +1,497 @@
+{
```
Comment (teoli2003) [inline javascript/grammar.json]: I think there are a few entries that are missing here:

null_literal for null
boolean_literals for true/false
decimal_numeric_literals
hexadecimal_numeric_literals
including the hexadecimal_escape_sequences
unicode_escape_sequences
array_literals
string_literals
regular_expression_literals

All with 'true' compat values.

**Comment by Elchi3** [comment]
Updated!

**Pull request merged**